README.md

```diff
@@ -303,12 +334,14 @@ See [third_party_integration](./third_party_integration) for examples of code mo
 - Ubuntu\* 18.04 or later (64-bit)
 - Python\* 3.7 or later
 - Supported frameworks:
-  - PyTorch\* 1.12.1
+  - PyTorch\* >=1.9.1, <1.14
   - TensorFlow\* >=2.4.0, <=2.11.1
```

Comment:
From setup.py - `tensorflow~=2.11.1`

Reply:
good catch! @alexsu52 , we should sort it out

Reply:
@KodiaqQ , I think in documents it's written correctly. setup.py should probably be updated but not in this PR

Reply:
agreed to fix it in develop by a separate PR